aptos-move/framework/aptos-framework/sources/configs/stake.move

```diff
@@ -674,6 +684,10 @@ module aptos_framework::stake {
         let old_locked_until_secs = stake_pool.locked_until_secs;
         stake_pool.locked_until_secs = new_locked_until_secs;
 
+        if (old_locked_until_secs <= timestamp::now_seconds()) {
+            stake_pool.rewards_minted = 0;
```

Comment:
This doesn't seem right. Why is this only updated here? The lockup could have expired and the validator might not call increase_lockup or increase_lockup_with_cap